### PR TITLE
Add catf & elk logs

### PR DESCRIPTION
**Pull request by DaveSys911**
**What does this PR do?**

1. Add logs aggregation from cloud beat & elk while tests are running.
```zsh
[pod/cloudbeat-n4k6t/cloudbeat] 2022-10-25T01:32:40.416759299Z {"log.level":"warn","@timestamp":"2022-10-25T01:32:40.416Z","log.logger":"elasticsearch","log.origin":{"file.name":"elasticsearch/client.go","file.line":429},"message":"Cannot index event publisher.Event{Content:beat.Event{Timestamp:time.Date(2022, time.October, 25, 1, 32, 39, 33918048, time.UTC), Meta:{\"index\":\"logs-cloud_security_posture.findings-default\"}, Fields:{\"agent\":{\"ephemeral_id\":\"1098c22b-2225-475c-a2fc-b016f7ce8111\",\"id\":\"f38c0063-8725-4359-bceb-b74f8b843ae7\",\"name\":\"kind-mono-control-plane\",\"type\":\"cloudbeat\",\"version\":\"8.6.0\"},\"cluster_id\":\"71b326f6-4917-493c-b866-04294bb4f5ca\",\"ecs\":{\"version\":\"8.0.0\"},\"event\":{\"category\":[\"configuration\"],\"created\":\"2022-10-25T01:32:39.033775382Z\",\"id\":\"810c2392-6c9a-42bb-940b-47dfdbf48603\",\"kind\":\"state\",\"outcome\":\"success\",\"sequence\":1666661557,\"type\":[\"info\"]},\"host\":{\"architecture\":\"aarch64\",\"containerized\":false,\"hostname\":\"kind-mono-control-plane\",\"ip\":[\"10.244.0.1\",\"10.244.0.1\",\"10.244.0.1\",\"172.18.0.2\",\"fc00:f853:ccd:e793::2\",\"fe80::42:acff:fe12:2\",\"10.244.0.1\",\"10.244.0.1\",\"10.244.0.1\"],\"mac\":[\"02:42:ac:12:00:02\",\"12:da:a6:a3:e6:aa\",\"2a:b0:57:cc:b6:5f\",\"5e:d6:d0:db:7f:85\",\"6a:31:fe:ac:08:fa\",\"7a:30:25:5d:88:11\",\"96:dc:5e:e2:af:a5\"],\"name\":\"kind-mono-control-plane\",\"os\":{\"codename\":\"bullseye\",\"family\":\"debian\",\"kernel\":\"5.10.104-linuxkit\",\"name\":\"Debian GNU/Linux\",\"platform\":\"debian\",\"type\":\"linux\",\"version\":\"11 (bullseye)\"}},\"message\":\"Rule \\\"Minimize wildcard use in Roles and ClusterRoles (Manual)\\\": passed\",\"resource\":{\"id\":\"3022bee8-e704-4c34-b477-2e4682d60bdf\",\"name\":\"cloudbeat-test\",\"raw\":{\"apiVersion\":\"rbac.authorization.k8s.io/v1\",\"kind\":\"ClusterRole\",\"metadata\":{\"annotations\":{\"meta.helm.sh/release-name\":\"cloudbeat-test\",\"meta.helm.sh/release-namespace\":\"kube-system\"},\"creationTimestamp\":\"2022-10-25T00:31:03Z\",\"labels\":{\"app.kubernetes.io/managed-by\":\"Helm\",\"date\":\"2022-10-25\",\"generator\":\"helm\",\"k8s-app\":\"cloudbeat-test\"},\"name\":\"cloudbeat-test\",\"resourceVersion\":\"92400\",\"uid\":\"3022bee8-e704-4c34-b477-2e4682d60bdf\"},\"rules\":[{\"apiGroups\":[\"\"],\"resources\":[\"nodes\",\"namespaces\",\"events\",\"pods\",\"services\"],\"verbs\":[\"get\",\"list\",\"watch\",\"create\",\"update\",\"patch\",\"delete\"]},{\"apiGroups\":[\"extensions\"],\"resources\":[\"replicasets\"],\"verbs\":[\"get\",\"list\",\"watch\"]},{\"apiGroups\":[\"apps\"],\"resources\":[\"statefulsets\",\"deployments\",\"replicasets\"],\"verbs\":[\"get\",\"list\",\"watch\"]},{\"apiGroups\":[\"batch\"],\"resources\":[\"jobs\"],\"verbs\":[\"get\",\"list\",\"watch\"]},{\"apiGroups\":[\"\"],\"resources\":[\"nodes/stats\"],\"verbs\":[\"get\"]},{\"nonResourceURLs\":[\"/metrics\"],\"verbs\":[\"get\"]}]},\"sub_type\":\"ClusterRole\",\"type\":\"k8s_object\"},\"result\":{\"evaluation\":\"passed\",\"evidence\":{\"cluster_roles\":{\"apiVersion\":\"rbac.authorization.k8s.io/v1\",\"kind\":\"ClusterRole\",\"metadata\":{\"annotations\":{\"meta.helm.sh/release-name\":\"cloudbeat-test\",\"meta.helm.sh/release-namespace\":\"kube-system\"},\"creationTimestamp\":\"2022-10-25T00:31:03Z\",\"labels\":{\"app.kubernetes.io/managed-by\":\"Helm\",\"date\":\"2022-10-25\",\"generator\":\"helm\",\"k8s-app\":\"cloudbeat-test\"},\"name\":\"cloudbeat-test\",\"resourceVersion\":\"92400\",\"uid\":\"3022bee8-e704-4c34-b477-2e4682d60bdf\"},\"rules\":[{\"apiGroups\":[\"\"],\"resources\":[\"nodes\",\"namespaces\",\"events\",\"pods\",\"services\"],\"verbs\":[\"get\",\"list\",\"watch\",\"create\",\"update\",\"patch\",\"delete\"]},{\"apiGroups\":[\"extensions\"],\"resources\":[\"replicasets\"],\"verbs\":[\"get\",\"list\",\"watch\"]},{\"apiGroups\":[\"apps\"],\"resources\":[\"statefulsets\",\"deployments\",\"replicasets\"],\"verbs\":[\"get\",\"list\",\"watch\"]},{\"apiGroups\":[\"batch\"],\"resources\":[\"jobs\"],\"verbs\":[\"get\",\"list\",\"watch\"]},{\"apiGroups\":[\"\"],\"resources\":[\"nodes/stats\"],\"verbs\":[\"get\"]},{\"nonResourceURLs\":[\"/metrics\"],\"verbs\":[\"get\"]}]}},\"expected\":null},\"rule\":{\"audit\":\"Retrieve the roles defined across each namespaces in the cluster and review\\nfor wildcards\\n```\\nkubectl get roles --all-namespaces -o yaml\\n```\\nRetrieve the cluster roles defined in the cluster and review for wildcards\\n```\\nkubectl get clusterroles -o yaml\\n```\\n\",\"benchmark\":{\"id\":\"cis_k8s\",\"name\":\"CIS Kubernetes V1.23\",\"version\":\"v1.0.0\"},\"default_value\":\"\",\"description\":\"Kubernetes Roles and ClusterRoles provide access to resources based on sets of objects and actions that can be taken on those objects. It is possible to set either of these to be the wildcard \\\"*\\\" which matches all items. Use of wildcards is not optimal from a security perspective as it may allow for inadvertent access to be granted when new resources are added to the Kubernetes API either as CRDs or in later versions of the product.\\n\",\"id\":\"8c46796a-9b8d-585b-8c01-48cde4eff2ec\",\"impact\":\"None.\\n\",\"name\":\"Minimize wildcard use in Roles and ClusterRoles (Manual)\",\"profile_applicability\":\"* Level 1 - Worker Node\\n\",\"rationale\":\"The principle of least privilege recommends that users are provided only the access required for their role and nothing more. The use of wildcard rights grants is likely to provide excessive rights to the Kubernetes API.\\n\",\"references\":\"\",\"remediation\":\"Where possible replace any use of wildcards in clusterroles and roles with specific objects or actions.\\n\",\"section\":\"RBAC and Service Accounts\",\"tags\":[\"CIS\",\"Kubernetes\",\"CIS 5.1.3\",\"RBAC and Service Accounts\"],\"version\":\"1.0\"}}, Private:interface {}(nil), TimeSeries:false}, Flags:0x0, Cache:publisher.EventCache{m:mapstr.M(nil)}} (status=400): {\"type\":\"mapper_parsing_exception\",\"reason\":\"failed to parse field [resource.raw.metadata.labels.app] of type [keyword] in document with id '8-bFDIQBLfW8JdXdMJhW'. Preview of field's value: '{kubernetes={io/managed-by=Helm}}'\",\"caused_by\":{\"type\":\"illegal_state_exception\",\"reason\":\"Can't get text on a START_OBJECT at 1:2996\"}}, dropping event!","service.name":"cloudbeat","ecs.version":"1.6.0"}
PID 75579 terminated with exit code 0
Tests Finished
NAME: cloudbeat-test
LAST DEPLOYED: Tue Oct 25 03:31:03 2022
NAMESPACE: kube-system
STATUS: deployed
REVISION: 1
TEST SUITE:     test-pod-v1
Last Started:   Tue Oct 25 04:31:52 2022
Last Completed: Tue Oct 25 04:32:40 2022
Phase:          Failed
NOTES:
1. Get the application URL by running these commands:
  export POD_NAME=$(kubectl get pods --namespace kube-system -l "app.kubernetes.io/name=k8s-cloudbeat-tests,app.kubernetes.io/instance=cloudbeat-test" -o jsonpath="{.items[0].metadata.name}")
  export CONTAINER_PORT=$(kubectl get pod --namespace kube-system $POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}")
  echo "Visit http://127.0.0.1:8080 to use your application"
  kubectl --namespace kube-system port-forward $POD_NAME 8080:$CONTAINER_PORT

POD LOGS: test-pod-v1
============================= test session starts ==============================
platform linux -- Python 3.9.15, pytest-7.1.2, pluggy-1.0.0 -- /usr/local/bin/python
cachedir: .pytest_cache
rootdir: /usr/src/app, configfile: pyproject.toml
plugins: xdist-2.5.0, order-1.0.1, forked-1.4.0, allure-pytest-2.9.45, dependency-0.5.1
collecting ... collected 31 items / 25 deselected / 6 selected

integration/tests/test_cloudbeat_integration.py::test_cloudbeat_pod_exist service_account created.
role created.
role created.
cluster_role created.
role_binding created.
role_binding created.
cluster_role_binding created.
config_map created.
daemon_set created. status='{'collision_count': None,
 'conditions': None,
 'current_number_scheduled': 0,
 'desired_number_scheduled': 0,
 'number_available': None,
 'number_misscheduled': 0,
 'number_ready': 0,
 'number_unavailable': None,
 'observed_generation': None,
 'updated_number_scheduled': None}'
PASSED
integration/tests/test_cloudbeat_integration.py::test_elastic_index_exists[file] PASSED
integration/tests/test_cloudbeat_integration.py::test_elastic_index_exists[process] PASSED
integration/tests/test_cloudbeat_integration.py::test_elastic_index_exists[k8s_object] PASSED
integration/tests/test_cloudbeat_integration.py::test_cloudbeat_pods_running PASSED
integration/tests/test_cloudbeat_integration.py::test_leader_election FAILED

=================================== FAILURES ===================================
_____________________________ test_leader_election _____________________________

fixture_data = ([{'api_version': None,
 'kind': None,
 'metadata': {'annotations': None,
              'cluster_name': None,
        ...03c2e142c62b'},
            'phase': None,
            'volumes_attached': None,
            'volumes_in_use': None}}])
elastic_client = <commonlib.elastic_wrapper.ElasticWrapper object at 0xffff9096ffa0>
cloudbeat_agent = Munch({'name': 'cloudbeat', 'namespace': 'kube-system', 'findings_timeout': 500})
k8s = <commonlib.kubernetes.KubernetesHelper object at 0xffff90942a00>

    @pytest.mark.pre_merge
    @pytest.mark.order(4)
    @pytest.mark.dependency(depends=["test_cloudbeat_pod_exist"])
    def test_leader_election(fixture_data, elastic_client, cloudbeat_agent, k8s):
        """
        This test verifies that k8s related findings are sent by a single agent
        :param fixture_data: (Pods list, Nodes list)
        :param elastic_client: Elastic API client
        :param cloudbeat_agent: Cloudbeat configuration
        :param k8s: Kubernetes client object
        :return:
        """
    
        query, sort = elastic_client.build_es_query(term={"type": "k8s_object"})
        pods, nodes = fixture_data
        leader_node = k8s.get_cluster_leader(
            namespace=cloudbeat_agent.namespace, pods=pods)
        assert leader_node != "", \
            "The Leader node could not be found"
    
        # Wait for all agents to send resources to ES
        res = wait_for_cycle_completion(elastic_client=elastic_client, nodes=nodes)
>       assert res, 'Not all nodes have completed a cycle within the configured threshold'
E       AssertionError: Not all nodes have completed a cycle within the configured threshold
E       assert False

integration/tests/test_cloudbeat_integration.py:109: AssertionError
==================================== PASSES ====================================
=========================== short test summary info ============================
PASSED integration/tests/test_cloudbeat_integration.py::test_cloudbeat_pod_exist
PASSED integration/tests/test_cloudbeat_integration.py::test_elastic_index_exists[file]
PASSED integration/tests/test_cloudbeat_integration.py::test_elastic_index_exists[process]
PASSED integration/tests/test_cloudbeat_integration.py::test_elastic_index_exists[k8s_object]
PASSED integration/tests/test_cloudbeat_integration.py::test_cloudbeat_pods_running
FAILED integration/tests/test_cloudbeat_integration.py::test_leader_election
================= 1 failed, 5 passed, 25 deselected in 46.73s ==================

Error: pod test-pod-v1 failed
pod "test-pod-v1" deleted
```

2. Pin kind versions to 1.23.12

3. Remove duplicate just commands

**Related Issues**

**Checklist**
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary README/documentation (if appropriate)

### CODE REVIEW

**Comment by DaveSys911** [comment]
@oren-zohar It should also fix our ci issues https://github.com/elastic/cloudbeat/actions/runs/3320801760

File: test-logs/.gitkeep [hunk 1]
```diff
@@ -0,0 +1 @@
+# Holder
```
Comment (oren-zohar) [inline test-logs/.gitkeep:1]: 💯

File: justfile [hunk 2]
```diff
@@ -118,46 +117,25 @@ load-pytest-eks:
...
-  #!/usr/bin/env bash
-  helm test {{TESTS_RELEASE}} -n {{NAMESPACE}} --kube-context kind-{{kind}} --timeout {{TESTS_TIMEOUT}} --logs 2>&1 | tee test.log
-  result_code=${PIPESTATUS[0]}
-  SUMMARY=$(cat test.log | sed -n '/summary/,/===/p')
-  echo "summary<<EOF" >> "$GITHUB_ENV"
-  echo "$SUMMARY" >> "$GITHUB_ENV"
-  echo "EOF" >> "$GITHUB_ENV"
-  exit $result_code
+run-tests target='default':
+  helm test {{TESTS_RELEASE}} -n {{NAMESPACE}} --kube-context kind-kind-multi --timeout {{TESTS_TIMEOUT}} --logs 2>&1 | tee {{TEST_LOGS_DIRECTORY}}/{{target}}-$(date +"%d-%m-%y-%H-%M-%S").log
```
Comment (oren-zohar) [inline justfile:130]: should we have the kind cluster name as a param? in case we don't want to use `kind-multi`?

Comment (DaveSys911) [reply justfile]: @oren-zohar Good catch, added it as a param.

**Pull request merged**